ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -400,6 +402,145 @@ def check_alba(logger):
                            'arakoon_connected')
 
     @staticmethod
+    @ExposeToCli('alba', 'disk-safety')
+    def get_disk_safety(logger):
+        """
+        Send disk safety for each vpool and the amount of namespaces with the lowest disk safety to DB
+        """
+        points = []
+        abms = []
+
+        test_name = 'disk-safety'
+        result = {
+                "repair_percentage": None,
+                "lost_disks": None
+            }
+        for service in ServiceHelper.get_services():
+            if service.type.name == ServiceType.SERVICE_TYPES.ALBA_MGR:
+                abms.append(service.name)
+
+        abms = list(set(abms))
+        abl = BackendHelper.get_albabackends()
+        for ab in abl:
+            # Determine if services are from ab instance
+            service_name = ServiceHelper.get_service(ab.abm_services[0].service_guid).name
+            if service_name not in abms:
+                continue
+
+            config = "arakoon://config/ovs/arakoon/{0}/config?ini=%2Fopt%2FOpenvStorage%2Fconfig%2Farakoon_cacc.ini".format(service_name)
+
+            # Fetch alba info
+            try:
+                namespaces = AlbaCLI.run('show-namespaces', config=config, to_json=True)[1]
+            except Exception as ex:
+                raise RuntimeError("Could not execute 'alba show-namespaces'. Got {0}".format(ex.message))
```

Comment:
What if the abm arakoons are down? The healthcheck just stops running? You should provide a `[EXCEPTION]` message to the customer.